definitions/syntetic/inc/docs-description.md

```diff
@@ -0,0 +1 @@
+some test
```

Comment:
```suggestion
# Test markdown file

This is a test markdown file.

Include it in your OpenAPI definition description like this: 


    description:
      $ref: path/to/file.md

```
I know it's just a test... but I think this spices it up a bit.  And if someone opens the file, they can see more about the behavior too.